packages/mds-logger/index.ts

```diff
@@ -217,6 +219,15 @@ async function error(...msg: any[]): Promise<any[]> {
   return censoredMsg
 }
 
+// eslint-disable-next-line @typescript-eslint/no-explicit-any
+async function log(logLevel: LogLevel, ...msg: any[]): Promise<any[]> {
+  return {
+    INFO: () => info(msg),
```

Comment:
It's possible you should re-spread the msg parts before passing to `info` et al. Would be interesting to see if the tests work as expected if you pass more than one element, e.g.
```
logger.log('WARN', 'This is a warning', { a: 1 }, { b: 2 })
```
Also, I think you can move one level of function call as follows:
```
async function log(logLevel: LogLevel, ...msg: any[]): Promise<any[]> {
  return {
    INFO: info,
    WARN: warn,
    ERROR: error
  }[logLevel](...msg)
}
```

Reply:
Nice!